doc/man3/SSL_get_ciphers.pod

```diff
@@ -63,10 +73,19 @@ free the return value itself.
 The stack returned by SSL_get1_supported_ciphers() should be freed using
 sk_SSL_CIPHER_free().
 
+The stack returned by SSL_bytes_to_cipher_list() should be freed using
+sk_SSL_CIPHER_free().
+
 =head1 RETURN VALUES
```

Comment:
Actually, ended up changing the return convention to use an output pointer, since we want to return SCSVs separately so that the state machine can act on them.